ai-models/train_model.py

```
@@ -4,22 +4,19 @@
 """
 
 import os
-import sys
 import pandas as pd
 import numpy as np
 import joblib
 from datetime import datetime, timedelta
-from typing import Dict, List, Tuple, Any
+from typing import Dict, Tuple, Any
```

Ruff (0.15.21)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)